components/service/glean/src/main/java/mozilla/components/service/glean/TimeUnit.kt

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.glean
+
+/**
+ * Enumeration of different resolutions supported by
+ * the Timespan metric type.
+ */
+enum class TimeUnit {
```

Comment:
Mh, this is a good idea. I was working under the wrong assumption that this would be internal only, but it isn't, since we need to use the type for generated files. I'll change it to `GleanTimeUnit` for now.

Reply:
On a second thought, do we really need to rename this? The objects lives in completely different namespaces and external callers don't really need to import our `TimeUnit`, only the generated files do. The collision only really happens in our library and it's constrained to the `TimespansStorageEngine`, which is the only one using them both.

Reply:
Fair enough, if it's meant for internal use only then it should be safe enough.